app-emulation/virt-manager/virt-manager-4.1.0.ebuild

```
@@ -102,4 +102,5 @@ pkg_postinst() {
 
 	optfeature "SSH_ASKPASS program implementation" lxqt-base/lxqt-openssh-askpass net-misc/ssh-askpass-fullscreen net-misc/x11-ssh-askpass
 	optfeature "QEMU host support" app-emulation/qemu[usbredir,spice]
+	optfeature "virt-install support" dev-libs/libisoburn
```

That doesn't seem right. virt-install works without `libisoburn` installed.

---

virt-install failed when libisoburn not installed.

see also: https://fars.ee/Zoh4
https://bugs.gentoo.org/846821

virt-install command:
sudo virt-install --debug --connect=qemu:///system --virt-type=kvm --name=centos7 --memory=2048 --vcpus=2 --cpu=host-passthrough --location=/home/chris/workspace/qemu/iso/CentOS-7-x86_64-Minimal-2009.iso --file=/home/chris/workspace/qemu/disk/centos.qcow2,format=qcow2 --network network=default,model=virtio --nographics --extra-args console=ttyS0

---

Yes, that *particular* command line might not have worked, but virt-install is definitely capable of working without `libisoburn`.

```suggestion
	optfeature "virt-install --location ISO support" dev-libs/libisoburn
```

---

Yes, I modified this part with your suggestion.